src/status_im/ui/screens/chat/styles/message/message.cljs

```diff
@@ -171,21 +171,8 @@
    :flex-direction :row-reverse})
 
 (defn message-view
-  [{:keys [content-type mentioned pinned]}]
+  [{:keys [content-type]}]
   (merge
-   {:border-radius 10}
-
-   (cond
-     pinned                                             {:background-color colors/pin-background}
```

Comment:
how pinned related to this PR ?

Reply:
Oh it's not related. I was working on the pins issue at the same time. These lines got removed anyways.